src/components/CardColor/CardColor.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link } from "react-router-dom"
+
+interface CardColorProps {
+  title: string
+  path: string
+  color: string
+}
+
+export const CardColor = (props: CardColorProps) => {
+  const { title, path, color } = props
+  return (
+    <Link to={path}>
+      <div
+        style={{ backgroundColor: color }}
```

Comment:
W naszym wypadku raczej nie chcemy podawać koloru przez propsa bo nie będą się zgadzać z naszym Tailwindem. Doinstaluj paczkę https://www.npmjs.com/package/classnames i zmiast `color` chcemy przekazać `className`. W linijce niżej będzie to wtedy wyglądać to mniej więcej tak className={classNames('m-10 ...', className <=== te z propsa)}